test/Cabinet.Tests/FileSystem/PathExtensionFacts.cs

```diff
@@ -69,8 +66,8 @@ public class PathExtensionFacts {
             Assert.Equal(isChildOf, actualResult);
         }
 
-        public static object[] GetTestPaths() {
-            return new object[] {
+        public static object[][] GetTestPaths() {
+            return new [] {
                 new object[] { @"c:\foo", @"c:", false, false },
```

Comment:
`Is_SubDirectory_Of` is failing only on the first `GetTestPaths()` item for me. Happening for both .NET versions.

Seems like it's being flagged as a subdirectory of C: which could either be a true statement or not depending on how you look at it. Seems like it might be reasonable to just change the test to allow this case?

Reply:
That's odd, all tests are passing for me and they're passing on AppVeyor.

Reply:
I wonder if it could somehow be environmental?

Reply:
I guess if it works generally we can ignore it.

Reply:
Troubling but not sure how to replicate

Reply:
Well that extension method is internal and used no where so I'm just going to remove it...